src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -2023,25 +2022,19 @@ RemoveDuplicateJoinRestrictions(JoinRestrictionContext *joinRestrictionContext)
 
 
 /*
- * JoinRestrictionListExistsInContext returns true if the given joinRestrictionInput
+ * ShouldAddJoinRestrictionToContext returns true if the given joinRestriction
  * has an equivalent of in the given joinRestrictionContext.
  */
 static bool
-JoinRestrictionListExistsInContext(JoinRestriction *joinRestrictionInput,
-								   JoinRestrictionContext *joinRestrictionContext)
+ShouldAddJoinRestrictionToContext(JoinRestriction *joinRestriction,
```

Comment:
we have  both `joinRestrictionContext ` and `contextJoinRestriction `. so maybe use `joinRestrictionInTheContext` instead of `contextJoinRestriction`

Also, instead of `joinRestriction ` maybe use `joinRestrictionInTest`